test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -3477,7 +3477,12 @@ def __init__(self, name=None, attrs={}, defaults={}, pbsconf_file=None,
         self.shortname = self.hostname.split('.')[0]
         self.platform = self.du.get_platform()
 
-        self.logutils = None
+        try:
```

Comment:
Initializing self.logutils on top of class not working since it causes circular dependencies between logutils() and PBSService(). Therefore we have to import  and initialize pbs_logutils inside _init_() of PBSService.  I have renamed it to self.lu for consistency

Reply:
That's too bad, thanks for renaming it.